src/server/contentSecurityPolicy.ts

```diff
@@ -197,6 +198,7 @@ const frameSrc = (() => {
 const fontSrc = (() => {
   const defaultFontSrc = [
     "'self'",
+    ' data:',
```

Comment:
Er dette dumt?

Reply:
Er nok ikke superfarlig.
Er en angrepsvektor som jeg tror er vanskelig å utnytte til noe fornuftig. Og det krever at angriperen kan kontrollere style e.l for å sette fonten.

Så jeg tipper det går greit. Hvorfor trenger vi det?

Reply:
Tja. Regnes fonter som kjørbare filer som kan utnyttes?

Reply:
Ikke på den måten. Men man kan jo injecte en font for å manipulere hva som står i tekster.
Så man kan potensielt endre hva som står rundt om på nettsiden. Krever et annet hull for å utnyttes da.

Reply:
Vi fikk en del kjeft i prod-buildet når jeg ikke inkluderte det. Av en eller annen grunn gjaldt det kun kinesisk font